src/outputs.ts

```diff
@@ -54,6 +20,8 @@ export interface BulkUploadReport {
  * @returns (BulkRedirectListItem[]) Raw redirect list entries for a CF Bulk Redirect List
  */
 export const makeBulkList = (input: RedirectProps[]): BulkRedirectListItem[] => {
+  const default_locale = DEFAULT_LOCALE || 'en-US'; // assume en-US
+  const collator = new Intl.Collator(default_locale, { sensitivity: 'base' });
```

Comment:
Make sure we're case-insensitive in the comparison of locale names (because the *standard* is en-US, but the tables and URIs want `en-us`.